test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -32,6 +37,27 @@ public MsbuildIntegrationTestFixture()
 
             var sdkPaths = Directory.GetDirectories(Path.Combine(_cliDirectory, "sdk"));
 
+            //Temporary patching process for System.Security.Cryptography.Pkcs.dll and deps.json files
+            //Will be removed when shipping
+            var patchDir = sdkPaths.Where(path => path.Split(Path.DirectorySeparatorChar).Last().StartsWith("5")).First();
+            TempPatching(patchDir);
+
+            string packagePath = null;
+            if (RuntimeEnvironmentHelper.IsWindows)
+            {
+                packagePath = Path.Combine(Environment.GetEnvironmentVariable("USERPROFILE"), ".nuget", "packages");
+            }
+            else
+            {
+                packagePath = Path.Combine(Environment.GetEnvironmentVariable("HOME"), ".local", "share", "NuGet", "v3-cache");
+            }
+
+            var pathCopyFrom = Path.Combine(packagePath, "system.security.cryptography.pkcs", "5.0.0-alpha1.19473.1", "lib", "netstandard2.1");
```

Comment:
I think there's an easier way to achieve this. 
You should make sure the assemblies are in the project output folder. Then from there you just get the current directory and just copy it.

Reply:
1. What's the project output folder?
2. Why would the assemblies be in the current directory?

Reply:
The assembly running this code itself will be in the bin folder. That is what I am referring to as the project output folder. 

I'd usually expect the assemblies to be in the current directory, yes. If they are not they can get copied.

Reply:
I think this is resolved. Please take a look!